internal/grid/handlers.go

```diff
@@ -56,7 +56,7 @@ const (
 	HandlerUpdateMetadata
 	HandlerWriteMetadata
 	HandlerCheckParts
-	HandlerRenameData
+	HandlerRenameData2
```

Comment:
Renaming will not change the ID. Put it at the end and keep `HandlerRenameData` - maybe as `HandlerRenameDataLegacy`.

Reply:
You are forcing me to make a checksum of all the handler names :D

Reply:
Yeah let me do that.